config/apiserver/deployment.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: hiveapi
+  namespace: hive
+  labels:
+    api: hiveapi
+    apiserver: "true"
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      api: hiveapi
+      apiserver: "true"
+  template:
+    metadata:
+      labels:
+        api: hiveapi
+        apiserver: "true"
+    spec:
+      containers:
+      # By default we will use the latest CI images published from hive master:
+      - image: registry.svc.ci.openshift.org/openshift/hivev1:hive
+        name: apiserver
+        resources:
+          requests:
+            cpu: 100m
+            memory: 100Mi
+          limits:
+            cpu: 100m
+            memory: 200Mi
+        command:
+          - "/opt/services/hive-apiserver"
+        args:
+          - "start"
+          - "--alsologtostderr"
```

Comment:
Out of curiosity, what do we get out of logging to file as well as stderr?

Reply:
none. :) i just copy-pasted the wrong CLI argument from the apiserver help output

Reply:
i went ahead and changed it to --logtostderr (as originally intended)